.prettierignore

```diff
@@ -0,0 +1 @@
+**
```

Comment:
Please, remove this file, this project doesn‘t use Prettier. It‘s up to each contributor to avoid that their tools/set up mess things with the code but this package should not hold those configurations.